ta/pkcs11/src/user_ta_header_defines.h

```diff
@@ -17,4 +17,13 @@
 #define TA_STACK_SIZE			(4 * 1024)
 #define TA_DATA_SIZE			(16 * 1024)
 
+#define TA_CURRENT_TA_EXT_PROPERTIES \
+    { "gpd.ta.description", USER_TA_PROP_TYPE_STRING, \
+        "PKCS#11 trusted application" }, \
+    { "gpd.ta.version", USER_TA_PROP_TYPE_U32, \
+      &(const uint32_t){ (PKCS11_TA_VERSION_MAJOR << 2) | \
```

Comment:
`gpd.ta.version` is of type String (GP spec v1.2 .1 Table 4-11).

Reply:
you're right. thanks.